shared-scripts.sh

```diff
@@ -451,6 +459,22 @@ function _deploy {
   ./codebuild_specs/scripts/publish.sh
 }
 
+function _deprecate {
+  loadCacheFromBuildJob
+  echo "Deprecate"
+  echo "Authenticate with NPM"
+  if [ "$USE_NPM_REGISTRY" == "true" ]; then
+      PUBLISH_TOKEN=$(echo "$NPM_PUBLISH_TOKEN" | jq -r '.token')
+      echo "//registry.npmjs.org/:_authToken=$PUBLISH_TOKEN" > ~/.npmrc
+  else
+    source codebuild_specs/scripts/local_publish_helpers.sh
+    startLocalRegistry "$(pwd)/codebuild_specs/scripts/verdaccio.yaml"
```

Comment:
Should we clear the Verdaccio cache first? I ran into some issues trying to do local workflow testing when I had an existing cache.